internal/audit/hub/filter.go

```diff
@@ -69,6 +70,29 @@ func NewAuditLogFilter(conf MaskConf) (*AuditLogFilter, error) {
 	}, nil
 }
 
+func NewOversizedLogFilter() (*AuditLogFilter, error) {
+	return NewAuditLogFilter(MaskConf{
```

Comment:
Unless I am mistaken, doesn't this strip out pretty much all the information from the message? I think that if we just strip `inputs[*].resource.attr`, `inputs[*].principal.attr` and `ouputs[*].filterDebug`, the chances are quite high that the message size would come back down to reasonable levels.